src/components/procedure/_app-findings-history.mdx

```diff
@@ -0,0 +1,5 @@
+To **view and add comments** to history of a finding:
+
+1. Click the <i className="fa-regular fa-arrow-up-right-from-square inline_svg"></i> **Finding details** icon next to the finding’s status.
+2. Find the comments in the **History** section.
```

Comment:
Can the screenshot here contain an actual history (ideally one from triaging by pr comment?)

Reply:
@raghavjain3 will add such a screenshot (note that this will be shared to Findings page documentation also, but I think that this is not an issue).